templates_test/insert.tpl

```diff
@@ -41,7 +41,11 @@ func test{{$tableNamePlural}}InsertWhitelist(t *testing.T) {
 
 	tx := MustTx(boil.Begin())
 	defer tx.Rollback()
+	{{if ne .DriverName "mssql"}}
 	if err = {{$varNameSingular}}.Insert(tx, {{$varNameSingular}}Columns...); err != nil {
+	{{- else -}}
+	if err = {{$varNameSingular}}.Insert(tx, {{$varNameSingular}}ColumnsWithoutDefault...); err != nil {
```

Comment:
Hm. We may make this the same for each database too then. I'll ask @nullbio on this one.